sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -323,7 +324,10 @@ public class SentryOptions {
   /**
    * Contains a list of origins to which `sentry-trace` header should be sent in HTTP integrations.
    */
-  private final @NotNull List<String> tracingOrigins = new CopyOnWriteArrayList<>();
+  private @Nullable List<String> tracePropagationTargets = null;
+
+  private final @NotNull List<String> defaultTracePropagationTargets =
+      Collections.singletonList(".*");
```

Comment:
Can we have just a list with the default value?

Reply:
Then we'd need to do checks like "does the list only contain the default item" in the add method. Or we only allow setting a list but not adding individually. This would cause problems with continuing to support the deprecated `tracingOrigins`.

Reply:
We don't need to check if it contains already, if it's duplicated, it's a user mistake (that won't have a side effect), we only need to make it mutable, either adding new cases or removing what's in there.

Reply:
As discussed earlier we're keeping it this way. It works as expected and due to the now deprecated `addTracingOrigin` method we still need to make sure users who are calling it only end up with entries they put in there and not the default value (that sends everywhere).